### PR TITLE
Changed the left image from img src to a background image

### PR DESCRIPTION
**Pull request by TurnbulA**
Put the image url into the style css
removed image from HTML

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -201,15 +201,14 @@ p{
   background-color: #f7f8f8;
   display: flex;
   justify-content: center;
+  height: 50rem;
 }
 .section-testimonial-image{
   width: 50%;
-}
+  background: url("https://picsum.photos/1000/900");
```
Comment (mali891) [inline style.css]: This can be cleaner written as:
`background: url() no-repeat center center;
background-size: cover;
`

Comment (TurnbulA) [reply style.css]: Just pushed that now

File: style.css [hunk 2]
```diff
@@ -205,8 +205,7 @@ p{
 }
 .section-testimonial-image{
   width: 50%;
-  background: url("https://picsum.photos/1000/900");
-  background-repeat: no-repeat;
+  background: url("https://picsum.photos/1000/900") no-repeat center center;
```
Comment (mali891) [inline style.css:208]: Does this also need `background-size: cover;` ?

Comment (TurnbulA) [reply style.css:208]: Pushed the addition of background-size: cover;

**Review by mali891 — APPROVED** [review]

**Pull request merged**